~/.local/log/stimmtausch doesn't exist on startup

This leads to

```
panic: open /home/makyo/.local/log/stimmtausch/stimmtausch.log: no such file or directory

goroutine 1 [running]:
main.main()
        /build/stimmtausch-lugPab/stimmtausch-0.0.2/obj-x86_64-linux-gnu/src/github.com/makyo/stimmtausch/cmd/st/main.go:25 +0x18e
```

--- a/client/connection.go
+++ b/client/connection.go
@@ -524,13 +524,13 @@ func NewConnection(name string, w config.World, s config.Server, cl *Client) (*c
 	}
 
 	log.Tracef("ensuring connection working directory")
-	if err := os.MkdirAll(c.getConnectionFile(""), 0755); err != nil {
+	if err := util.EnsureDir(c.getConnectionFile("")); err != nil {
 		log.Errorf("unable to ensure connection directory! %v", err)
 		return nil, err
 	}
 
 	log.Tracef("ensuring world log directory")
-	if err := os.MkdirAll(c.getLogFile(""), 0755); err != nil {
+	if err := util.EnsureDir(c.getLogFile("")); err != nil {
 		log.Errorf("unable to ensure log directory! %v", err)
 		return nil, err
 	}

--- a/cmd/st/main.go
+++ b/cmd/st/main.go
@@ -9,6 +9,7 @@
 package main
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
 
@@ -16,10 +17,14 @@ import (
 
 	"github.com/makyo/stimmtausch/cmd"
 	"github.com/makyo/stimmtausch/config"
+	"github.com/makyo/stimmtausch/util"
 )
 
 func main() {
 	config.InitDirs()
+	if err := util.EnsureDir(config.LogDir); err != nil {
+		panic(err)
+	}
 	f, err := os.Create(filepath.Join(config.LogDir, "stimmtausch.log"))
 	if err != nil {
 		panic(err)
@@ -28,4 +33,5 @@ func main() {
 	loggo.ReplaceDefaultWriter(loggo.NewSimpleWriter(f, loggo.DefaultFormatter))
 
 	cmd.Execute()
+	fmt.Fprint(os.Stderr, "\n")
 }

--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+stimmtausch (0.0.2ubuntu1) bionic cosmic; urgency=medium
+
+  * Expanding distrobutions for PPA. 
+
+ -- Madison Scott-Clary <makyo@drab-makyo.com>  Thu, 21 Feb 2019 09:42:00 -0800
+
 stimmtausch (0.0.2) bionic; urgency=medium
 
   * Adds ability to connect to multiple worlds at once, switch between

--- a/util/ensure_dir.go
+++ b/util/ensure_dir.go
@@ -1,0 +1,11 @@
+package util
+
+import (
+	"os"
+)
+
+// EnsureDir ensures that a directory is in place before opening a file there.
+func EnsureDir(path string) error {
+	log.Tracef("ensuring %s exists", path)
+	return os.MkdirAll(path, 0755)
+}

